core/components/babel/elements/snippets/babeltranslation.snippet.php

```diff
@@ -41,28 +40,27 @@
     return;
 
 /* get snippet properties */
-$resourceId = intval($modx->getOption('resourceId', $scriptProperties));
-if (empty($resourceId)) {
-    if (!empty($modx->resource) && is_object($modx->resource)) {
-        $resourceId = $modx->resource->get('id');
-    } else {
-        return;
-    }
+$resourceIds = $modx->getOption('resourceId', $scriptProperties);
+if (empty($resourceIds)) {
+    return;
 }
+$resourceIds = array_map('trim', explode(',', $resourceIds));;
 $contextKey = $modx->getOption('contextKey', $scriptProperties, '', true);
 if (empty($contextKey)) {
     $cultureKey = $modx->getOption('cultureKey', $scriptProperties, '', true);
     $contextKey = $babel->getContextKey($cultureKey);
 }
 $showUnpublished = $modx->getOption('showUnpublished', $scriptProperties, 0, true);
 
-/* determine id of tranlated resource */
-$linkedResources = $babel->getLinkedResources($resourceId);
-$output          = null;
-if (isset($linkedResources[$contextKey])) {
-    $resource = $modx->getObject('modResource', $linkedResources[$contextKey]);
-    if ($resource && ($showUnpublished || $resource->get('published') == 1)) {
-        $output = $resource->get('id');
+/* determine ids of translated resource */
+$output = array();
+foreach($resourceIds as $resourceId) {
+    $linkedResource = $babel->getLinkedResources($resourceId);
```

Comment:
The $linkedResources variable is missing an 's' here at the end..

Reply:
Thanks, but since that would be a single resource, it should be without an 's' later on.

Reply:
Fixed in https://github.com/mikrobi/babel/pull/176/commits/4316aa7e51d086a541760c435b869e79fc11b8d9